drydock/templates/kustomized/tutor16/base/apps/openedx/settings/partials/common_lms.py

```diff
@@ -0,0 +1,52 @@
+{% include "apps/openedx/settings/partials/common_all.py" %}
+
+######## Common LMS settings
+LOGIN_REDIRECT_WHITELIST = ["{{ CMS_HOST }}"]
+
+# Better layout of honor code/tos links during registration
+REGISTRATION_EXTRA_FIELDS["terms_of_service"] = "required"
```

Comment:
```suggestion
REGISTRATION_EXTRA_FIELDS["terms_of_service"] = "hidden"
```
From https://github.com/overhangio/tutor/commit/2d6b9c17674ebcc1ebd8cc4f56c3b0413c171300